### ISSUE TITLE
Learning rate and learning rate scheduler

### ISSUE THREAD

**Issue by dtdo90** [lyuwenyu/RT-DETR#531]
Dear author,

Thank you for the amazing work!!!

I have a question regarding the choice of learning rate and learing-rate scheduler. I the optimizer.yaml file, it set lr=0.0001

lr_scheduler:
  type: MultiStepLR
  milestones: [1000]
  gamma: 0.1

Is there a reason for setting such a small learning rate? Furthermore, the scheduler reduces learning rate at steps 1000 which is pretty early in the training?

**Comment by lyuwenyu** [comment]
Yes, It means we keep fixed lr in training process.

**Comment by SebastianJanampa** [comment]
@lyuwenyu 
Why did you not use a learning rate drop as in previous detr models?
